Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -85,6 +85,14 @@ struct SettingsView: View {
                             Text(vehicle.make)
                             
                             Text(vehicle.model)
+
+                            Text(vehicle.year)
```

Comment:
Because these are now optional nil coalescing is needed
I'll make a different issue to make this look better